x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc

```diff
@@ -65,8 +71,11 @@ To set up the watch:
     }
   },
 --------------------------------------------------
+// NOTCONSOLE
+--
 . Load data into the watch payload by creating an input that searches the meetup data for events that have _Open Source_ as a topic. You can use aggregations to group the data by city, consolidate references to the same events, and sort the events by date.
 +
```

Comment:
leftovers or asciidoc features I am not aware off?

Reply:
If you mean the + and -- around the code example, it's just a way to ensure it stays nested in the list. Not really necessary in this case.